### PR TITLE
formatted and resolved files

### PR DESCRIPTION
**Pull request by IamDikshaR**

### CODE REVIEW

File: src/atoms/headings/five/two-xl/semibold/index.astro
Comment (tanishdotdev) [inline src/atoms/headings/five/two-xl/semibold/index.astro:1]: Why have you named it two-xl if the text size used is 3xl?

File: src/atoms/headings/six/xl/semibold/index.astro
Comment (tanishdotdev) [inline src/atoms/headings/six/xl/semibold/index.astro:1]: Fix the formatting. This should be formatted in 3 lines.

File: src/atoms/headings/two/six-xl/medium/index.astro
Comment (tanishdotdev) [inline src/atoms/headings/two/six-xl/medium/index.astro:1]: Rename it correctly, text used is 8xl, you have named it six-xl.

File: src/atoms/lists/list-item/index.astro
Comment (tanishdotdev) [inline src/atoms/lists/list-item/index.astro:1]: list item atom can be multiple so add text-size to the path.
src/atoms/lists/list-item/two-xl/index.astro

File: src/components/home/solid-light-system/index.astro
Comment (tanishdotdev) [inline src/components/home/solid-light-system/index.astro:1]: 1. Remove semicolons.
2. Format the imports in ascending order.
3. Create an array of list item and map it.
4. Create atom for video.
5. Format the map() like this.
Screenshot 2024-07-16 at 12 26 23 PM

File: src/components/home/solid-light-system/title-description/index.astro
Comment (tanishdotdev) [inline src/components/home/solid-light-system/title-description/index.astro:1]: 1. Fix formatting in props.
2. Remove semicolons.
3. Fix formatting like adding spaces in { title }

File: src/components/home/testimonials/index.astro
Comment (tanishdotdev) [inline src/components/home/testimonials/index.astro:1]: Create atom for buttons.

File: src/components/home/the-tech/index.astro
Comment (tanishdotdev) [inline src/components/home/the-tech/index.astro:1]: 1. Remove the semicolons in all files.
2. Format the imports in ascending order.
3. Fix formatting in map().
4. Add comments in js.
5. Fix spacing in curly brackets like this { data.title }

File: src/layouts/layout/index.astro
Comment (tanishdotdev) [inline src/layouts/layout/index.astro:1]: Don't change anything in layout.

File: src/components/home/welcome/index.astro
Comment (tanishdotdev) [inline src/components/home/welcome/index.astro:1]: 1. Remove semicolons.
2. Format props.

**Review by tanishdotdev — CHANGES_REQUESTED** [review]
The Formatting is incorrect in all files. It should be 4 spaces not 8.

File: src/atoms/headings/five/two-xl/semibold/index.astro
Comment (IamDikshaR) [reply src/atoms/headings/five/two-xl/semibold/index.astro:1]: It was altered and i missed it. Fixed it now.

File: src/atoms/headings/two/six-xl/medium/index.astro
Comment (IamDikshaR) [reply src/atoms/headings/two/six-xl/medium/index.astro:1]: It was altered and i missed it. Fixed it now.

File: src/atoms/headings/six/xl/semibold/index.astro
Comment (IamDikshaR) [reply src/atoms/headings/six/xl/semibold/index.astro:1]: done

File: src/atoms/lists/list-item/index.astro
Comment (IamDikshaR) [reply src/atoms/lists/list-item/index.astro:1]: done

File: src/layouts/layout/index.astro
Comment (IamDikshaR) [reply src/layouts/layout/index.astro:1]: linked material icons

File: src/components/home/solid-light-system/index.astro
Comment (IamDikshaR) [reply src/components/home/solid-light-system/index.astro:1]: done

File: src/components/home/testimonials/index.astro
Comment (IamDikshaR) [reply src/components/home/testimonials/index.astro:1]: done

File: src/components/home/solid-light-system/title-description/index.astro
Comment (IamDikshaR) [reply src/components/home/solid-light-system/title-description/index.astro:1]: done

File: src/components/home/the-tech/index.astro
Comment (IamDikshaR) [reply src/components/home/the-tech/index.astro:1]: done

**Pull request closed without merging**